Libraries/process/Tests/test_DefaultLauncher.cpp

```diff
@@ -0,0 +1,43 @@
+/**
+ Copyright (c) 2015-present, Facebook, Inc.
+ All rights reserved.
+
+ This source code is licensed under the BSD-style license found in the
+ LICENSE file in the root directory of this source tree. An additional grant
+ of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+#include <gtest/gtest.h>
+#include <process/DefaultLauncher.h>
+#include <process/LaunchResult.h>
+#include <process/MemoryContext.h>
+#include <libutil/DefaultFilesystem.h>
+
+#include <unistd.h>
+
+using process::DefaultLauncher;
+using process::MemoryContext;
+using libutil::DefaultFilesystem;
+
+TEST(DefaultLauncher, CapturesStdout)
+{
+    DefaultLauncher launcher;
+    auto filesystem = DefaultFilesystem::GetDefaultUNSAFE();
```

Comment:
Hmm, I think the issue is that you're trying to test the real (`Default`) implementations. Those are inherently not isolated since they are the bridge to the real OS, so they'd need to be integration tests. I think for now it might be best just not to test it until some kind of integration test infrastructure is in place?